controllers/topicsController.js

```diff
@@ -0,0 +1,13 @@
+const { selectTopics } = require("../models/topicsModel");
+
+const getTopics = (req, res, next) => {
+  selectTopics()
+    .then((topics) => {
+      res.status(200).send({ topics });
+    })
+    .catch((err) => {
```

Comment:
What error are you anticipating for this end point? There isn't really a way this could go wrong so isn't required.